sdk/ml/azure-ai-ml/azure/ai/ml/entities/_builders/base_node.py

```diff
@@ -566,7 +566,10 @@ def _refine_optional_inputs_with_no_value(cls, node: "BaseNode", kwargs):
         :param kwargs: The kwargs
         :type kwargs: dict
         """
-        for key, value in node.inputs.items():
+        inputs: Dict[str, Any]
```

Comment:
Any change `Dict[str, object]` would be acceptable typing here? It's _almost_ the same thing as `Dict[str, Any]` except mypy doesn't allow arbitrary operations without type guards.

If you're familiar with typescript, it's basically the same as `any` vs `unknown`.

See: https://mypy.readthedocs.io/en/stable/dynamic_typing.html#any-vs-object